packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/ExoPlayerEventListener.java

```diff
@@ -14,11 +14,16 @@ final class ExoPlayerEventListener implements Player.Listener {
   private final ExoPlayer exoPlayer;
   private final VideoPlayerCallbacks events;
   private boolean isBuffering = false;
-  private boolean isInitialized = false;
+  private boolean isInitialized;
 
   ExoPlayerEventListener(ExoPlayer exoPlayer, VideoPlayerCallbacks events) {
+    this(exoPlayer, events, false);
+  }
+
+  ExoPlayerEventListener(ExoPlayer exoPlayer, VideoPlayerCallbacks events, boolean wasInitialized) {
```

Comment:
Maybe there is a design guide somewhere that disagrees, but doesn't `was` imply in the past and (usually) no longer? is `is` not more appropriate? Not a huge deal either way.

Reply:
Yeah I think this was a poor word choice on my part, I'd be happy to rename this `isInitialized` or even `initialized` for consistency. Let me do that before merging.

Reply:
Done!